modules/components/pages/outputs/quest_db.adoc

```diff
@@ -0,0 +1,423 @@
+= questdb
+:type: output
+:status: experimental
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
```

Comment:
```suggestion
// © 2024 Redpanda Data Inc.
```

Is this still required? All doc pages include it at the bottom of the page already and the year is calculated on each build. Embedding it in each page like this requires extra maintenance overhead.

Reply:
As this is a site wide change, I've created an issue for it: https://redpandadata.atlassian.net/browse/DOC-640